README.md

```diff
@@ -16,24 +16,35 @@ To see a current list of commands, run `make help`.
 
 `UNPAYWALL_EMAIL`: email address to include in API call as required in their [documentation](https://unpaywall.org/products/api). Your personal email is appropriate for development. Deployed and for tests, use the timdex moira list email.
 
-#### Authentication
+### Optional
+
+`PLATFORM_NAME`: The value set is added to the header after the MIT Libraries logo. The logic and CSS for this comes
+from our theme gem.
+
+### Authentication
+
+#### Required in all environments
 
 Access to some of the config values below is limited. Please contact someone in the EngX team if you need help locating
 them.
 
 `BASE_URL`: The base url for the app. This is required for Omniauth config.
-`FAKE_AUTH_CONFIG`: Switches Omniauth to developer mode when set to `true`. Fake auth is also enabled whenever the
-app is started in the development environment, or if the app is a PR build (see `HEROKU_APP_NAME` under optional
-variables).
-`OP_HOST`: The OID provider hostname, required for authentication. (Do not include URL prefix.)
-`OP_SECRET_KEY`: The secret key for the OID client.
-`OP_CLIENT_ID`: The identifier for the OID client.
-`OP_ISSUER`: The URL for the OIDC issuer. This can be found in the Touchstone OpenID metadata.
-
-### Optional
-
-`HEROKU_APP_NAME`: Used by the FakeAuthConfig module to determine whether an app is a PR build.
-`PLATFORM_NAME`: The value set is added to the header after the MIT Libraries logo. The logic and CSS for this comes from our theme gem.
+`OPENID_HOST`: The OID provider hostname, required for authentication. (Do not include URL prefix.)
+`OPENID_SECRET_KEY`: The secret key for the OID client.
+`OPENID_CLIENT_ID`: The identifier for the OID client.
+`OPENID_ISSUER`: The URL for the OIDC issuer. This can be found in the Touchstone OpenID metadata.
+
+#### Required in PR builds
+
+The config below is needed to run Omniauth in developer mode in Heroku review apps. Rather than relying upon a single
+ENV value, we use the `FakeAuthConfig` module to perform additional checks that confirm whether developer mode should
+be enabled. This assures that developer mode is never enabled in staging or production apps.
+
+`FAKE_AUTH_ENABLED`: Switches Omniauth to developer mode when set. If unset, PR builds will attempt to authenticate with
+OIDC, which will fail as their domains are not registered with the provider. (Note: Developer mode is also enabled
+whenever the app is started in the development environment.)
+`HEROKU_APP_NAME`: Used by the FakeAuthConfig module to determine whether an app is a PR build. If this is set along
```

Comment:
This still isn't clear that nobody should actually ever set `HEROKU_APP_NAME` because it happens automatically.

Maybe:

`HEROKU_APP_NAME`: This is set automatically in PR builds and is used by the FakeAuthConfig module to determine whether an app is a PR build. This should never be manually set or overridden in any environment as it is not necessary to do that and can only cause bad things to happen.

Reply:
Yes, good call. I added a note along those lines.